libp2p/Host.cpp

```diff
@@ -783,15 +807,21 @@ void Host::run(boost::system::error_code const& _ec)
     unsigned reqConn = 0;
     {
         RecursiveGuard l(x_sessions);
-        for (auto const& p : m_peers)
         {
```

Comment:
these braces are redundant

Reply:
Thanks, got confused due to `DEV_RECURSIVE_GUARDED `use in some places